sklearn/inspection/tests/test_partial_dependence.py

```diff
@@ -45,6 +46,11 @@
 multioutput_regression_data = (make_regression(n_targets=2, random_state=0), 2)
 
 
+def teardown_module(module):
+    if has_matplotlib():
+        close_figure()
```

Comment:
So does that mean we don't need to call `close_figure()` in all the tests of this module anymore?

If that's the case, I would suggest changing the defualt of `close_figure()` from `None` to `'all'` for it to be more obvious.

Reply:
> So does that mean we don't need to call close_figure() in all the tests of this module anymore?

This is the idea.

> If that's the case, I would suggest changing the defualt of close_figure() from None to 'all' for it to be more obvious.

We can do that.

Reply:
if we do, we need this to be merged before the release